README.md

```diff
@@ -3,3 +3,8 @@
 | Route | Description | Parameters | Expected Output |
 |-------|-------------|------------|-----------------|
 | `POST /api/login`| logs the user in | body: `{ email: string, password: string}` | `{ token: string, refresh_token: string }` |
+| `POST /api/user?=1/create`| Create a list | body `{ title: string, description: string}` | `{ title: string, description: string}` |
```

Comment:
this path is wrong, it wont work. You should take a look at REST pattern. This path would be: `POST /api/user/{id}`
where {id} is a parameter

Reply:
| `POST /api/user/task` | Create a List | body `{ title: string, description: string}` | `{ title: string, description: string}` |